resources/eventing/values.yaml

```diff
@@ -45,9 +45,37 @@ global:
       statusPort: 15020
       portName: status
 
-  securityContext:
-    allowPrivilegeEscalation: false
+  podSecurityContext:
+    fsGroup: 10001
+    runAsUser: 10001
+    runAsGroup: 10001
+    runAsNonRoot: true
+    seccompProfile:
+      type: RuntimeDefault
+    sysctls:
+      - name: kernel.shm_rmid_forced
+        value: ''
+      - name: net.ipv4.ip_local_port_range
+        value: ''
+      - name: net.ipv4.ip_unprivileged_port_start
+        value: ''
+      - name: net.ipv4.tcp_syncookies
+        value: ''
+      - name: net.ipv4.ping_group_range
+        value: ''
+
+  containerSecurityContext:
```

Comment:
Is NATS already running as non-root user? would now it would work with all NATS commands? storage etc.

Reply:
Yes, it is verified offline.